test/TestAppUtils/TestFrame.cs

```diff
@@ -124,6 +143,19 @@ protected override void OnApplyTemplate()
             _unhandledExceptionReportingTextBox = (TextBox)GetTemplateChild("UnhandledExceptionReportingTextBox");
         }
 
+        private void ToggleThemeButton_Click(object sender,RoutedEventArgs e)
+        {
+            if (CurrentTheme == ElementTheme.Light)
```

Comment:
Is there a reason why we cannot just do this?
_rootGrid.RequestedTheme = (_rootGrid.RequestedTheme == ElementTheme.Light) ? ElementTheme.Dark : ElementTheme.Light;

Reply:
If we do not update the `CurrentTheme` property, the app would switch would switch to the default theme upon navigation to a TestPage since `CurrentTheme` would always be `ElementTheme.Default`.

Reply:
How about checking for ElementTheme.Default, and if that's the case you can switch it to the opposite of whatever the ApplicationTheme is?

Reply:
Great idea! I refactored the code and was able to get rid of the "RequestedTheme" field.

Reply:
Awesome, looks good!